<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signuproom</class>
 <widget class="QWidget" name="signuproom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignUp</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="lineEdit_ID_SignUp">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Psw">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CPsw">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SignUp">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SignUp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_FindPsw">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Find Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID_FindPsw">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sign UP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Find Password:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
